src/components/Mealplans/MealplanCard.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Card, Button } from "react-bootstrap";
+import DefaultImg from "../../assets/images/spencer-davis-_yWhM5TZgdo-unsplash.jpg";
+import { useHistory } from "react-router-dom";
+import { DateTime } from "luxon";
+
+export const MealplanCard = ({ mealplan }) => {
+  let history = useHistory();
+
+  const redirectToUpdateMealplan = () => {
+    history.push(`/user/update-mealplan/${mealplan._id}`);
+  };
+
+  const startDate = DateTime.fromISO(mealplan.start_date);
+  const endDate = DateTime.fromISO(mealplan.end_date);
+
+  const buildRecipeList = (recipes) => {
+    return recipes.length ? (
+      <ul>
+        {" "}
+        {recipes.map((recipe) => {
+          return (
+            <li key={recipe._id}>
+              Day {recipe.day} - {recipe.recipe.title}
+            </li>
+          );
+        })}{" "}
+      </ul>
+    ) : (
+      "No recipes in this mealplan"
+    );
+  };
+
+  return (
+    <Card className="mx-0 mealplan-card">
+      <Card.Img className="mealplan-img" src={DefaultImg} />
+      <Card.Body className="mealplan-body">
+        <Card.Title>{mealplan.title}</Card.Title>
+        <Card.Text>
+          Duration: {mealplan.duration} | Start Date: {startDate.toLocaleString()} | End Date:{" "}
```

Comment:
could we add 'days' after duration so it readys '7 days' rather than '7' or use a ternary and output '1 week' or '2 weeks'